ui/pages/permissions-connect/connect-page/connect-page.tsx

```diff
@@ -98,8 +98,10 @@ export const ConnectPage: React.FC<ConnectPageProps> = ({
     ? [...nonTestNetworks, selectedTestNetwork].map(({ chainId }) => chainId)
     : nonTestNetworks.map(({ chainId }) => chainId);
 
+  const allNetworksList = [...nonTestNetworks, ...testNetworks].map(({ chainId }) => chainId)
+
   const supportedRequestedChainIds = requestedChainIds.filter((chainId) =>
-    selectedNetworksList.includes(chainId),
+    allNetworksList.includes(chainId),
```

Comment:
this is a bug in main. should probably hoist this up

Reply:
I'll PR this separately

Reply:
Alex addressed this here https://github.com/MetaMask/metamask-extension/pull/30139